Decrease the fontSize for the payment button in the checkout to ensure the text fits within the button (GCOM-1186)

carlocarels90 opened the pull request:
| Before | After |
|--------|--------|
| ![localhost_3000_checkout_payment(iPhone 12 Pro) (1)](https://github.com/graphcommerce-org/graphcommerce/assets/62989724/f52c2127-2b60-444a-9e84-39d135384506) | ![localhost_3000_checkout_payment(iPhone 12 Pro) (2)](https://github.com/graphcommerce-org/graphcommerce/assets/62989724/b7c23d6f-3c2d-4ba2-9c16-84954722483b) |

paales commented:
@carlocarels90 Might it be a better idea to remove the payment method name on the button. It was a 'cute' idea, but making the text smaller doesn't look like a great idea?

carlocarels90 commented:
> @carlocarels90 Might it be a better idea to remove the payment method name on the button. It was a 'cute' idea, but making the text smaller doesn't look like a great idea?

@paales I think you're right. Adding other payment methods with even longer names in the future won't work either. I've reverted the font sizes and removed the payment method from the button.

paales commented:
@carlocarels90 Can you rebase the branch with canary?

carlocarels90 commented:
> @carlocarels90 Can you rebase the branch with canary?

@paales done! ✅

paales commented on .changeset/real-garlics-battle.md:
```diff
@@ -0,0 +1,5 @@
+'@graphcommerce/magento-cart-payment-method': patch
+---
+
+decrease the fontSize for the payment button in the checkout to ensure the text fits within the button
```
```suggestion
Remove the selected payment method title from the Place order button so that the text always fits
```
